### ISSUE TITLE
More than one account should be able to use the same telephone number.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [vector-im/element-meta#893]
### Additional context

https://github.com/matrix-org/matrix-spec/issues/1400#issue-1529655555

**Comment by RokeJulianLockhart** [comment]
Is this limitation additionally true for e-mail addresses?

**Comment by t3chguy** [comment]
> Is this limitation additionally true for e-mail addresses?

Yes given that e-mails can be used for password reset, so you have to be able to resolve a single account to reset the password on from an e-mail address on a given Matrix server. You can use the same e-mail and phone number on multiple Matrix servers

**Comment by RokeJulianLockhart** [comment]
@t3chguy, I've used accounts online which had my telephone number in different accounts. When performing password-resets, they merely allowed me to choose the account that the telephone number was attached to. This is no problem because obviously all of those accounts are owned by me. This is also true for e-mail.

The current limitation is a grave because it means that my accounts must become less secure due to them being forced to use different credentials.

**Comment by t3chguy** [comment]
@rokejulianlockhart, sure, this just isn't something Matrix supports at this time, if you want this to change you may wish to open an issue with https://github.com/matrix-org/matrix-spec/issues

**Comment by RokeJulianLockhart** [comment]
@t3chguy, I'll report it there. Thanks.
